service/rest_version.go

```diff
@@ -149,15 +149,20 @@ func (restapi restAPI) getRecentVersions(w http.ResponseWriter, r *http.Request)
 			return
 		}
 	}
-
-	// add one to limit to determine if a new page is necessary
-	versions, err := model.VersionFind(model.VersionBySystemRequesterOrdered(projectId, start).
-		Limit(l + 1).WithoutFields(model.VersionConfigKey))
-	if err != nil {
-		msg := fmt.Sprintf("Error finding recent versions of project '%v'", projectId)
-		grip.Error(errors.Wrap(err, msg))
-		gimlet.WriteJSONInternalError(w, responseError{Message: msg})
-		return
+	var versions []model.Version
+	projectId, err := model.FindIdForProject(projectIdentifier)
+	// only look for versions if the project can be found, otherwise continue without error
+	if err == nil {
+		// add one to limit to determine if a new page is necessary
+		fmt.Printf("project %s can be found with id %s\n", projectIdentifier, projectId)
```

Comment:
This writes to stdout. Was this debug code?

Reply:
ummm good point, probably